### PR TITLE
[FIX] LGTM.com warning: Variable defined multiple times

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
This assignment to '`cv_scores`' is unnecessary as it is redefined [`here`](https://lgtm.com/projects/g/nilearn/nilearn/snapshot/3af46e7cd3a9362bb815e3deb17d8c3d95485758/files/examples/03_connectivity/plot_extract_regions_dictlearning_maps.py?sort=name&dir=ASC&mode=heatmap#x673dd6739f275dd8:1) before this value is used.

https://lgtm.com/projects/g/nilearn/nilearn/snapshot/3af46e7cd3a9362bb815e3deb17d8c3d95485758/files/examples/07_advanced/plot_advanced_decoding_scikit.py#x5b739e019ffb38d3:1

### CODE REVIEW

File: examples/07_advanced/plot_advanced_decoding_scikit.py [hunk 1]
```diff
@@ -85,9 +85,10 @@
 # the function :func:`sklearn.model_selection.cross_val_score` that computes
 # for you the score for the different folds of cross-validation.
 from sklearn.model_selection import cross_val_score
+# Here `cv=5` stipulates a 5-fold cross-validation
 cv_scores = cross_val_score(svc, fmri_masked, conditions, cv=5)
+print("Scikit-learn accuracy: {:.3f}".format(cv_scores.mean()))
```
Comment (bthirion) [inline examples/07_advanced/plot_advanced_decoding_scikit.py:90]: Maybe rather: SVC accuracy ?

Comment (DimitriPapadopoulos) [reply examples/07_advanced/plot_advanced_decoding_scikit.py:90]: True. But we need to distinguish between these two prints::
* https://github.com/DimitriPapadopoulos/nilearn/blob/d99c6d8/examples/07_advanced/plot_advanced_decoding_scikit.py#L90
* https://github.com/DimitriPapadopoulos/nilearn/blob/d99c6d8/examples/07_advanced/plot_advanced_decoding_scikit.py#L109

Perhaps:
```python
print("SVC accuracy: {:.3f}".format(cv_scores.mean()))
```
```python
print("SVC accuracy (tuned parameters): {:.3f}".format(cv_scores.mean()))
```

Comment (DimitriPapadopoulos) [reply examples/07_advanced/plot_advanced_decoding_scikit.py]: Done.

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks!

**Comment by NicolasGensollen** [comment]
Merging, thanks!
